roles/reset/tasks/lxc.yml

```diff
@@ -0,0 +1,61 @@
+---
+- name: Set rc.local fact
+  set_fact:
+    rc_local: "{{ lookup('template', 'templates/rc.local.j2') }}"
+
+- name: Remove rc.local modificaitons for proxmox lxc containers
+  become: true
+  blockinfile:
+    path: /etc/rc.local
+    content: '{{ rc_local }}'
+    create: false
+    state: absent
+
+- name: check for container files that exist on this host
```

Comment:
since this is created by a separate role, should we have this in a separate task file?